blog/r-tidyfinance-0-1-0/index.qmd

```diff
@@ -0,0 +1,185 @@
+```
+Under the hood, the function uses the `frenchdata` package and applies some cleaning steps, as in TFWR. If you haven't installed `frenchdata` yet, you'll get prompted to install it first before you can download this specific data type. 
+
+You can also access q-Factor data in this way, by calling:
+
+```{r}
+download_data("factors_q5_daily", "2020-01-01", "2020-12-31")
+```
+
+To ensure that we can extend the functionality of the download functions for specific types, we also provide domain-specific download functions. The `download_data("factors_ff3_monthly")` actually calls `download_data_factors("factors_ff3_monthly")`, which in turn calls `download_data_factors_ff("factors_ff3_monthly")`. Why did we decide to have these nested function approach?
+
+Suppose that the q-Factor data changes its URL path and our original function does not work anymore. In this case, you can replace the default `url` value in `download_data_factors_q(type, start_date, end_date, url)` to apply the usual cleaning steps. 
+
+This feature becomes more apparent for other data sources such as `wrds_crsp_monthly`. Note that you need to have valid WRDS credentials and need to set them correctly (check `?get_wrds_connection` and [WRDS, CRSP, and Compustat](../../r/wrds-crsp-and-compustat) in TFWR). If you want to download the standard monthly CRSP data, you can call:
+
+```{r}
+#| output: false
+#| eval: false
+download_data("wrds_crsp_monthly", "2020-01-01", "2020-12-31")
+```
+
+However, if you want to add further columns, you can add them via `...` to `download_data_wrds_crsp()`, for instance:
+
+```{r}
+#| output: false
+#| eval: false
+download_data_wrds_crsp("wrds_crsp_monthly", "2020-01-01", "2020-12-31", mthvol)
+```
+
+Similarly, if you want to download the old version of CRSP, you can use the `version` parameter in `download_data_wrds_crsp()` (see this blog post here TODO: INSERT LINK). 
+
+As another example, you can do the same for Compustat:
+```{r}
+#| output: false
+#| eval: false
+download_data_wrds_compustat("wrds_compustat_annual", "2000-01-01", "2020-12-31", acoxar, amc, aldo)
+```
+
+Check out the list of supported types and the corresponding download functions for more information on the respective customization options. We decided to provide limited functionality for the initial release on purpose and rather respond to community request than overengineer the package from the start.  
+
+## Browse content 
+
+We include functions to check out content from tidy-finance.org. If you want to list all available R chapters, simply call the following function:
+
+```{r}
+list_tidy_finance_chapters()
+```
+
+The function returns a character vector containing the names of the chapters available in TFWR. If you want to look at a specific chapter, you can call:
+
+```{r}
+#| eval: false
+open_tidy_finance_website("beta-estimation")
+```
+
+This opens a specific website or the main index page in you default web broswer. 
```

Comment:
"your" default web browser.
"This opens either the specific chapter you requested or the main index page. "